src/vellum/workflows/events/workflow.py

```diff
@@ -169,6 +172,21 @@ def outputs(self) -> OutputsType:
     def final_state(self) -> Optional[StateType]:
         return self.body.final_state
 
+    @field_serializer("body")
+    def serialize_body(
+        self, body: WorkflowExecutionFulfilledBody[OutputsType, StateType], info: SerializationInfo
+    ) -> WorkflowExecutionFulfilledBody[OutputsType, StateType]:
```

Comment:
consolidate with the `serialize_body` method defined in the initiated event body, by hoisting it up to _BaseWorkflowEvent